packages/run-protocol/test/vaultFactory/test-liquidator.js

```diff
@@ -454,13 +454,13 @@ const makeDriver = async (t, initialPrice, priceBase) => {
     checkPayouts: async (expectedRUN, expectedAEth) => {
       const payouts = await E(currentSeat).getPayouts();
       const collProceeds = await aeth.issuer.getAmountOf(payouts.Collateral);
-      const runProceeds = await E(run.issuer).getAmountOf(payouts.RUN);
+      const runProceeds = await E(run.issuer).getAmountOf(payouts.Minted);
       t.deepEqual(runProceeds, expectedRUN);
       t.deepEqual(collProceeds, expectedAEth);
     },
     checkRewards: async expectedRUN => {
```

Comment:
s/expectedRUN/expectedWanted/

Reply:
the test uses RUN brand so I think this is still valid, just as `expectedAEth` is